### PR TITLE
Create Readme

### PR DESCRIPTION
**Pull request by Shaktisinh81**
3-4 day in add now botton

### CODE REVIEW

**Comment by Shaktisinh81** [comment]
Botton apply

**Comment by Shaktisinh81** [comment]
<script>
      paypal.Buttons({
        // Order is created on the server and the order id is returned
        createOrder() {
          return fetch("/my-server/create-paypal-order", {
            method: "POST",
            headers: {
              "Content-Type": "application/json",
            },
            // use the "body" param to optionally pass additional order information
            // like product skus and quantities
            body: JSON.stringify({
              cart: [
                {
                  sku: "YOUR_PRODUCT_STOCK_KEEPING_UNIT",
                  quantity: "YOUR_PRODUCT_QUANTITY",
                },
              ],
            }),
          })
          .then((response) => response.json())
          .then((order) => order.id);

**Comment by Shaktisinh81** [comment]
Apply to hi

**Comment by Shaktisinh81** [comment]
gh pr checkout 55

**Comment by Shaktisinh81** [comment]
<script> paypal.Buttons({ // Order is created on the server and the order id is returned createOrder() { return fetch("/my-server/create-paypal-order", { method: "POST", headers: { "Content-Type": "application/json", }, // use the "body" param to optionally pass additional order information // like product skus and quantities body: JSON.stringify({ cart: [ { sku: "YOUR_PRODUCT_STOCK_KEEPING_UNIT", quantity: "YOUR_PRODUCT_QUANTITY", }, ], }), }) .then((response) => response.json()) .then((order) => order.id);

File: standard-integration/Readme [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+            // use the "body" param to optionally pass additional order information
+            // like product skus and quantities
+            body: JSON.stringify({
+              cart: [
+                {
+                  sku: "YOUR_PRODUCT_STOCK_KEEPING_UNIT",
+                  quantity: "YOUR_PRODUCT_QUANTITY",
+                },
+              ],
+            }),
```
Comment (MALIKSHAHID786) [inline standard-integration/Readme:20]: Helo sir

Comment (MALIKSHAHID786) [reply standard-integration/Readme:20]: Have anice dau

**Pull request closed without merging**